lib/govuk_elements_form_builder/form_builder.rb

```diff
@@ -38,13 +38,17 @@ def add_error_to_html_tag! html_tag
       end
     end
 
+    def form_group_id attribute
+      group_id = "error_#{@object_name}_#{attribute}" if error_for? attribute
```

Comment:
The `group_id` variable isn't used. So we don't need the `group_id =`. If we get rubocop setup soon, it'll raise these sort of issues. :)